### DISCUSSION TITLE
StreetComplete asks for Information, that Osmand already knows

### DISCUSSION THREAD

**Discussion by Tommy-01** [streetcomplete/StreetComplete#4239] [Q&A] [answered]
I thought the Android Osmand Navigator App uses the OpenStreetMaps?
But StreetComplete asks for Information, that already available in the Osmand App (Like the Name of a Street, or opening time of a shop).

Can anybody explain?  Maybe i am just wrong.
Thank you!

**Comment by Helium314** [comment, marked as answer]
StreetComplete regularly asks for opening hours, as they may change rather frequently. But in this case the current opening are shown, which you probably would have noticed.

Can you show a street / shop where this is happening?
